awesome-list-guidelines.md

```diff
@@ -0,0 +1,254 @@
+<!-- omit from toc -->
+# Awesome List Guidelines
+
+> [!NOTE]
```

Comment:
There are 4 notes and 1 tip in this section. Please consider adding moving some of the content to generic text for ease of reading.